cpu/cortexm_common/Makefile.include

```diff
@@ -31,6 +31,18 @@ LINKFLAGS += $(if $(ROM_OFFSET),$(LINKFLAGPREFIX)--defsym=_rom_offset=$(ROM_OFFS
 # FW_ROM_LEN: rom length to use for firmware linking. Allows linking only in a section of the rom.
 LINKFLAGS += $(if $(FW_ROM_LEN),$(LINKFLAGPREFIX)--defsym=_fw_rom_length=$(FW_ROM_LEN))
 
+# Size of the AUX slot in Bytes - must align with flash page size
+SLOT_AUX_LEN ?= 0
+SLOT_AUX_OFFSET ?= $(shell echo $$(($(ROM_LEN:%K=%*1024)-$(SLOT_AUX_LEN))))
```

Comment:
Since AUX slot requires linker support, it is the right place for not to define it only `cpu/cortexm_common`.

Reply:
As of now riotboot/slot is only implemented for `cortexm_common`. I had hoped that any new implementation could just add the needed line

```python
rom_aux  (rx)   : ORIGIN = _rom_start_addr + _slot_aux_offset, LENGTH = _slot_aux_len
```

to the linker script so this could still live in a common place.